src/components/form-helpers/form-helpers.js

```diff
@@ -281,6 +283,7 @@ const _PhoneFieldItem = ({
   placeholder,
   required,
   formik: { values, handleChange, handleBlur, setFieldValue },
+  dependencies: { ipinfoClient },
```

Comment:
the second `i` dont need to be in uppercase? like `ipInfoClient` ?

Reply:
I prefered keep it in lowercase because the api's domain es ipinfo.io